src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -63,7 +65,7 @@ public TypedParameter(string name, ProtoCore.Type type, AssociativeNode defaultV
         /// <summary>
         /// Returns fully qualified string representation of AST node.
         /// </summary>
-        [JsonProperty("DefaultValue")]
+        [JsonIgnore]
```

Comment:
We want to keep this one as-is.  The support for it is in a CoGS branch right now.

Reply:
Sure. Let me remove this `Ignore`

Reply:
It needs the old value put back, not just the Ignore removed.